custom_components/tahoma/__init__.py

```diff
@@ -151,6 +151,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 
     platforms = defaultdict(list)
     platforms[SCENE] = scenarios
+    platforms["sensor"] = []
```

Comment:
No constant?

Reply:
You can just add the sensors platform manually once we retrieved all the available platforms.

Reply:
> Do you plan to clean the state_attributes method?
> 
> https://github.com/iMicknl/ha-tahoma/blob/669ac3f1f5448e521cdd2223a6226bb06b319045/custom_components/tahoma/tahoma_entity.py#L80

I was thinking about removing this fully eventually, especially since this is not allowed in core. However, in the mean-time it can help us spot attributes / states that we should add.